download.py

```diff
@@ -47,7 +47,7 @@ def get_correctly_labeled_entries(all_entries):
 
 
 def download_csv_dataset(csv_url, dataset_csv_path):
-    logging.info("Downloading CSV from ", csv_url, " to ", dataset_csv_path, ".")
+    logging.info("Downloading CSV from %s to %s.", csv_url, dataset_csv_path)
```

Comment:
```suggestion
    logging.info(f"Downloading CSV from {csv_url} to {dataset_csv_path}.")
```
We're moving to using f-strings in all the codebase.

Reply:
Or use the `.format()` method.